docker/CMSRucioClient/scripts/updateDDMQuota

```diff
@@ -90,28 +81,35 @@ def calculate_ddm_quotas():
         if static == 0:
             continue  # Skip if static is 0
 
+        # Apply penalty for special rses
+        rse_attributes = client.list_rse_attributes(rse)
+        if "ddm_quota_penalty" in rse_attributes:
```

Comment:
I see two problems with this suggestion:

1. It wouldn't be the same thing if you mean 
```
        static, rucio, expired = (x for x in (static, rucio, expired))
        free = static - rucio
        ddm_quota = static_weight * (static / total_static) + free_weight * \
                    (free / static) + expired_weight * (expired / static)
        if "ddm_quota_penalty" in rse_attributes:        
            ddm_quota *= float(rse_attributes["ddm_quota_penalty"])
```
It yields different values.

2. If we do it as you suggest, it becomes harder to explain what we're doing. We explain `ddm_quota_penalty` as the factor which reduces all space metrics of an RSE, e.g. if CERN has 100PB of static, 10PB of free, we'll treat it as if has 50PB of static and 5PB of free if the penalty value is 0.5. I don't know how to explain yours.

Reply:
A penalty should not be used to say `reduce the actual values to lower` rather to `reduce the factor`. 
So, you say, we use a smaller coefficient (as a penalty) compared to rest. 
And not that I shrink the rse to a smaller one - that would not be clear actually - we aren't reducing the site space just giving it less weight. 

Yes, it would end up in different numbers, given how the relative weights are calculated.

Reply:
In the light of latest discussions and your comment, I suggest two things:
1. rename "ddm_quota" to "dm_weight". This will be used for tape placements only for now. 

2. I suggest removing ddm_quota_penalty attribute and replace it with "override_factor" attribute which should be applied as follows:
```
dm_weight =  (STATIC_WEIGHT * <relative-pledge> + FREE_WEIGHT * <relative-free-space>) * <override_factor>
```
With the current status quo, we can set the "override_factor" to a value more than 1 for FNAL Tape and a value lower than 1 for CERN Tape. Does it sounds reasonable? @dynamic-entropy